README.md

```diff
@@ -247,6 +248,49 @@
 <hr>
 
 
+
+# ROS2 LifeCycleNode
+
+The `USE_LIFECYCLE_NODES` cmake flag enables **ROS2 Lifecycle Nodes** (`rclcpp_lifecycle::LifecycleNode`) in the **Realsense SDK**, providing better node management and explicit state transitions.  
+
+However, enabling this flag introduces a limitation where **Image Transport functionality (`image_transport`) is** <span style="color:#ff6666">**disabled**</span> **when `USE_LIFECYCLE_NODES=ON`**.  
+This means that **compressed image topics (e.g., JPEG, PNG, Theora) will not be available** and<br>
+**Subscribers** must use raw image topics, which may increase bandwidth usage.
+
+> Note: Users who do not depend on image_transport will not be affected by this change and can safely enable Lifecycle Nodes without any impact on their workflow.
+
+### 📌 Why Does This Limitation Exist?
+The `image_transport` package does **not support Lifecycle Nodes** (`rclcpp_lifecycle::LifecycleNode`).  

```

Comment:
I think it may be a bit too much information, maybe we should write that in the stage where we add lifecycle node support , image_transport did not support it